app/src/main/res/layout/multi_select_speed_dial.xml

```diff
@@ -7,6 +7,8 @@
         android:id="@+id/fabSDOverlay"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:backgroundTint="@color/white"
+        android:alpha="0.30"
```

Comment:
`SpeedDialOverlayLayout` sets its background internally. Are you sure that this is needed?

Reply:
Ok, will do.

Not sure, it is strange. I don't remember removing the tint and don't know where in the history find it. I ended up resorting to this because I don't know another way. Maybe you could help?

Reply:
Hmm, weird. Apparently the first screen on which the fab is shown works and all subsequent screens have no background. Your PR does not fix this.